.ci/.docker-images.yml

```diff
@@ -0,0 +1,389 @@
+---
+# Dockerfile build config for .github/workflows/build-docker-images.yml
+# Schema: .ci/.docker-images.schema.json
+templates:
+
+  - &apm-agent-library-docker-images
+    repository: "elastic/apm-pipeline-library"
+    working_directory: ".ci/docker"
+
+  - &opbeans-docker-images
+    tag: "daily"
+
+  - &apm-agent-python-docker-images
+    name: "apm-agent-python"
+    repository: "elastic/apm-agent-python"
+    working_directory: "tests"
+
+  - &apm-agent-nodejs-docker-images
+    name: "apm-agent-nodejs"
+    repository: "elastic/apm-agent-nodejs"
+    working_directory: ".ci/docker/node-container"
+
+  - &apm-agent-ruby-docker-images
+    name: "apm-agent-ruby"
+    repository: "elastic/apm-agent-ruby"
+    working_directory: "spec"
+
+  - &apm-agent-rum-docker-images
+    repository: "elastic/apm-agent-rum-js"
+
+  - &observability-dev-docker-images
+    repository: "elastic/observability-dev"
+
+#  - &beats-docker-images
+#    # Install gvm (https://github.com/moovweb/gvm#installing)
+#    prepare_script: |
+#      bash < <(curl -s -S -L https://raw.githubusercontent.com/moovweb/gvm/master/binscripts/gvm-installer)
+#      . $HOME/.gvm/scripts/gvm
+#      GO_VERSION="go$(cat ${GITHUB_WORKSPACE}/.go-version)"
+#      gvm install go1.4 -B
+#      gvm use go1.4
+#      export GOROOT_BOOTSTRAP=$GOROOT
+#      gvm install "${GO_VERSION}"
+#      gvm use "${GO_VERSION}"
+#      GOBIN="${GOROOT}/bin"
+#      echo "${GOBIN}" >> $GITHUB_PATH
+#      echo "GOBIN=${GOBIN}" >> $GITHUB_ENV
+#      echo "GOROOT=${GOROOT}" >> $GITHUB_ENV
+#      echo "GOPATH=${GOPATH}" >> $GITHUB_ENV
+#    build_script: "make mage && mage compose:buildSupportedVersions"
+#    push_script: "make mage && mage compose:pushSupportedVersions"
+
+images:
+
+  # beats Docker Images
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/metricbeat-integrations-images-x-pack/
+#  - <<: *beats-docker-images
+#    name: "metricbeat-integrations-images"
+#    repository: "elastic/beats"
+#    working_directory: "metricbeat"
+
+## This is also failing in Jenkins
+## https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/metricbeat-integrations-images/
+#  - <<: *beats-docker-images
+#    name: "metricbeat-integrations-images-x-pack"
+#    repository: "elastic/beats"
+#    working_directory: "x-pack/metricbeat"
+
+  # observability-test-environments Docker Images
+
+  - name: "functional-opbeans"
+    repository: "elastic/observability-test-environments"
+    working_directory: "tests"
+    build_script: "docker build --force-rm -t ${REGISTRY}/${PREFIX}/${NAME}:${TAG} ${NAME}"
+    push_script: "docker push ${REGISTRY}/${PREFIX}/${NAME}:${TAG}"
+
+## This is also failing in Jenkins
+##  https://apm-ci.elastic.co/job/apm-shared/job/docker-images/job/oracle-instant-client/
+#  - name: "azure-vm-tools"
+#    repository: "elastic/azure-vm-extension"
+#    working_directory: ".ci/docker/azure-vm-tools"
```

Comment:
I think is not used , TBC

Reply:
Should we keep it commented for now?

Reply:
Could you check if there are references on any repo? if not, I think it is fine to delete it.

Reply:
This image is used here: https://github.com/elastic/azure-vm-extension/blob/4ef62d7fa2d47873797df828c3903a3f8ce03858/.ci/packer_cache.sh

Reply:
Will be handled in https://github.com/elastic/apm-pipeline-library/issues/1992